ehr2meds/generate_synthetic_raw_data.py

```
@@ -0,0 +1,179 @@
+
+        # Get columns
+        linked_cols = linked_df[linked_on]
+
+        if linked_type == "choice":
+            selected_idx = random.randint(0, len(linked_cols) - 1)
+            selected_row = linked_cols.iloc[[selected_idx]].copy()
+        else:
+            raise ValueError(f"Unknown linked type: {linked_type}")
+        # Insert column to row
+        if rename_to:
+            selected_row = selected_row.rename(columns=dict(zip(linked_on, rename_to)))
```

Ruff (0.15.10)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)